folium/plugins/timeline.py

```diff
@@ -0,0 +1,289 @@
+from typing import List, Optional, TextIO, Union
+
+from branca.element import MacroElement
+from jinja2 import Template
+
+from folium.elements import JSCSSMixin
+from folium.features import GeoJson
+from folium.folium import Map
+from folium.utilities import JsCode, camelize, get_bounds, parse_options
+
+
+class Timeline(GeoJson):
+    """
+      Creates a layer from GeoJSON with time data to append
+      into a map with Map.add_child.
+
+      To add time data, you need to do one of the following:
+
+      * Add a 'start' and 'end' property to each feature. The start and end
+        can be any comparable item.
+
+      Alternatively, you can provide
+      a `get_interval` function.
+
+      * This function should be a JsCode object and take as parameter
+        a GeoJson feature and return a dict containing values for
+        'start', 'end', 'startExclusive' and 'endExcusive' (or false if no
+        data could be extracted from the feature).
+      * 'start' and 'end' can be any comparable items
+      * 'startExclusive' and 'endExclusive' should be boolean values.
+
+      Parameters
+      ----------
+      data: file, dict or str.
+          The geojson data you want to plot.
+
+      get_interval: JsCode
```

Comment:
```suggestion
      get_interval: JsCode, optional
```

Reply:
You marked this as resolved, but looks like the docstring didn't change? `get_interval` is an optional parameter isn't it?